prepare_source

```diff
@@ -1 +1,22 @@
-apt_src runc
+version_orig="1.3.0"
+version="1.3.0-1"
+debian_version="${version_orig}${version_suffix}"
+message="New upstream version ${version_orig}"
+
+SALSA_REPO_COMMIT="4fe73a4a8390da812ced05ccbb42f960035d91c2"
+
+git_src_commit v${version_orig} https://github.com/opencontainers/runc.git
+
+runc_salsa=$(mktemp -d)
+git clone --depth 1 https://salsa.debian.org/go-team/packages/runc $runc_salsa
+pushd $runc_salsa
+git sparse-checkout set debian
+git fetch origin $SALSA_REPO_COMMIT
+git checkout $SALSA_REPO_COMMIT
```

Comment:
`debian_version` has been the tag you could just `git clone`. As it's human readable instead of the commit hash I recommended using it. If you want to use the commit hash anyway please just add a comment to it, that refers to the version tag of salsa and remove `debian_version`.